cli/command/image/testdata/remove-command-success.Image Deleted with force option.golden

```diff
@@ -0,0 +1,2 @@
+NotFound:image1
```

Comment:
Was discussing with @vdemeester that the naming of these files is a bit odd; perhaps we should normalise them to be lowercase and have spaces replaced with `-`. Given that these files are generated, and this is in line with other files in this tests, we can keep it for now 👍